app/assets/javascripts/application.js.erb

```diff
@@ -194,9 +194,9 @@ function notificationAlertClose(){
 
 $(document).ready(function(){
   $('.tree-link a').each( function(){
-    truncateLongString( $(this), 30);
+    truncateLongString( $(this), <%= Constants::NAME_TRUNCATION_LENGTH %>);
```

Comment:
True, but then we have duplicate code in Ruby and JS. It doesn't seem clean to me...